tests/test_java.py

```diff
@@ -281,3 +284,21 @@ def test_async_profiler_stops_after_given_timeout(
 
         ap_proc.status_async_profiler()
         assert "Profiler is not active\n" in cast_away_optional(ap_proc.read_output())
+
+
+@pytest.mark.parametrize("in_container", [True])
+@pytest.mark.parametrize("image_suffix", ["_j9"])
+def test_sanity_j9(
```

Comment:
Any reason not to merge this with `test_java_async_profiler_cpu_mode`? I am pretty sure that async-profiler works in cpu mode for J9 as well.

Reply:
No reason to merge, we're not saving on tests. These tests 2 different things so they should remain 2 different tests